portfolio/src/main/webapp/style.css

```diff
@@ -2,17 +2,74 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  text-align: center;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* intro section */
+h1 {
+    margin-left: 30px;
```

Comment:
Ensure indentation is consistent throughout the file; this file started with a 2-space indent, I'd keep using that.